### PR TITLE
Use sistent schemas for design/filter and fix redirection link of docs

### LINKED ISSUE #10691: Broken docs link in Meshery UI
If I am importing a design type of Helm in MeshMap, what is it expecting? I tried chart URL, chart.yaml and nothing worked.
Also the URL referenced in the help, <https://docs.meshery.io/guides/meshery-design> is a 404. Is there an updated link for this? <#C02F90W866T|meshery-extensions>

[Slack Message](https://layer5io.slack.com/archives/C02F90W866T/p1712582115077219?thread_ts=1712582115.077219&cid=C02F90W866T)

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #10691 and related to #10445 
1. The link on tooltip for docs was not clickable and one place the link was also not correct which is fixed here: https://github.com/layer5io/sistent/pull/561
2. While i was here i used the available scehmas from sistent for import designs and filters and publish designs and filters
3. Add notification on successful filter and design import #10562 (1)

 -  ### Acceptance test for above fix:
- [x] Remove existing API calls for getting schemas
- [x] Use Sistent library to import schemas
- [x] No change should be made in UI or where ever we are using schemas, if required give reason why

Schema updated for:

- [x] Import filter
- [x] publish filter
- [x] import design
- [x] publish design

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Filters.js [hunk 1]
```diff
@@ -1468,13 +1474,13 @@ const ImportModal = React.memo((props) => {
...
-  const { publishFormSchema, handleClose, handlePublish, title } = props;
+  const { handleClose, handlePublish, title } = props;
 
   return (
     <Modal
       open={true}
-      schema={publishFormSchema.rjsfSchema}
-      uiSchema={publishFormSchema.uiSchema}
+      schema={publishCatalogItemSchema}
+      uiSchema={publishCatalogItemUiSchema}
```
Comment (betelgeuse-code) [inline ui/components/Filters.js:1483]: thankyou for this, do make sure you remove the API code for fetching schema, also make sure that this schema goes through the function that publishFormSchema.rjsfSchema goes, we don't get meshmodels already in schema we update them in UI

Comment (betelgeuse-code) [reply ui/components/Filters.js:1483]: For technology field

File: ui/components/MesheryPatterns.js [hunk 2]
```diff
@@ -1816,14 +1822,14 @@ const ImportModal = React.memo((props) => {
...
+  const { handleClose, handleSubmit, title } = props;
 
   return (
     <>
       <Modal
         open={true}
-        schema={publishFormSchema.rjsfSchema}
-        uiSchema={publishFormSchema.uiSchema}
+        schema={publishCatalogItemSchema}
+        uiSchema={publishCatalogItemUiSchema}
```
Comment (betelgeuse-code) [inline ui/components/MesheryPatterns.js:1832]: Same here

**Review by betelgeuse-code — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
Thanks @betelgeuse-code for the feedback, i will do so

**Comment by sudhanshutech** [comment]
@Yashsharma1911 i have updated, you can test this now

File: ui/package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
         "@fortawesome/free-regular-svg-icons": "^6.1.2",
         "@fortawesome/free-solid-svg-icons": "^6.4.0",
         "@fortawesome/react-fontawesome": "^0.2.0",
-        "@layer5/sistent": "^0.14.18",
+        "@layer5/sistent": "^0.14.21",
```
Comment (leecalcote) [inline ui/package.json:42]: Bump this to 0.14.23?

**Comment by sudhanshutech** [comment]
@Yashsharma1911 looks good?

**Comment by leecalcote** [comment]
merge conflicts... 

Gents, where do we stand on this PR? Still WIP or have we tossed in the towel?

**Comment by 11happy** [comment]
@leecalcote I have taken reference from @sudhanshutech PRs as told by him & have also included further work in my PR #10720

**Comment by leecalcote** [comment]
@11happy ok. Based on your description, I assume this PR is no longer being advanced.

**Pull request closed without merging**